main/round.js

```diff
@@ -0,0 +1,60 @@
+const Frame = require('./frame')
+// Uncomment line below for node demo
+// const showBreakdown = require('./showbreakdown')
+
+class Round {
```

Comment:
Good SRP skills separating these classes, using dependency injection on this interaction could be a good way of isolating them even more.